docs/project_defn.rst

```diff
@@ -254,10 +251,16 @@ on the `GCP Batch <https://cloud.google.com/batch>`_ service.
       repository.
 *  ``job_environment``: Optional. Specifies the computing requirements for each simulation.
 
-    * ``vcpus``: Number of CPUs needed. Default: 1.
-    * ``memory_mib``: Amount of RAM memory needed for each simulation in MiB. Default: 1024. For large multifamily buildings
-      this works better if set to 2048.
-    * ``machine_type``: GCP Compute Engine machine type to use. If omitted, GCP batch will choose a machine type based on the requested vCPUs and memory.
+    * ``vcpus``: Number of CPUs to allocate for running each simulation. Default: 1.
```

Comment:
Most (but not all 😞) of the bullets in the rest of the doc put two spaces after the bullet.

Reply:
Updated.